tests/test_postfix.py

```diff
@@ -0,0 +1,33 @@
+"""This module contains the tests for the postfix container, the image with postfix, sendmail and mailq pre-installed."""
+
+from bci_tester.data import POSTFIX_CONTAINERS
+
+CONTAINER_IMAGES = POSTFIX_CONTAINERS
+
+
+def test_postfix_status(auto_container):
+    # check if Postfix is running inside the container.
+    assert "localhost" in auto_container.connection.check_output(
+        "echo ${SERVER_HOSTNAME}"
+    )
+
+    assert (
+        "the Postfix mail system is running"
+        in auto_container.connection.check_output(
+            "postfix status > /tmp/status_output.txt 2>&1; cat /tmp/status_output.txt"
```

Comment:
```suggestion
            "postfix status 2>&1"
```

should work the same way

Reply:
should we actually check that there is *no other* output, like an error message or warning beforehand? and I think that the PID is always 1? can we check for that as well?

Reply:
addressed the  `postfix status 2>&1` code change suggestion in latest refresh. Thank you!

> and I think that the PID is always 1? can we check for that as well?

Here, do you mean to check if `ENTRYPOINT` command is running at PID 1? (if yes, I added a assertion for that)

> should we actually check that there is no other output, like an error message or warning beforehand?
Pending. I'll send changes.